src/app/features/collectibles/hooks/use-get-ordinals-query.ts

```diff
@@ -0,0 +1,36 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { useCurrentBtcAccountAddressIndexZero } from '@app/store/accounts/blockchain/bitcoin/bitcoin-account.hooks';
+
+import { ordApiXyzGetInscriptionByAddressSchema } from '../types';
+
+export function useGetOrdinalsQuery() {
```

Comment:
Should this just live with our other queries in `query` folder?

Reply:
It can, what's the convention or criteria for where queries are placed?

Reply:
Currently, we put them all under `src/app/queries`

Reply:
Ok, moved :) I saw `utils` files sprinkled throughout the queries folder, so created one for the ordinal validator, let me know if this works?